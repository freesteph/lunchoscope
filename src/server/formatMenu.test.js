const formatMenu = require("./formatMenu");

const mockData = {
  meat: { description: "chicken", price: "3" },
  veg: { description: "spinash", price: "2" },
  sides: { description: "fries & gravy", price: "1" }
};

describe("formatMenu", () => {
  it("should format the menu entry properly", () => {
    expect(formatMenu(mockData)).toMatchSnapshot();
  });
});
