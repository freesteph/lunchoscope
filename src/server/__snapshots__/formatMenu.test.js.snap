// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatMenu should format the menu entry properly 1`] = `
"Today's menu is:
:poultry_leg: chicken - *3*
:green_salad: spinash - *2*
:fries: fries & gravy - *1*"
`;
